importer/importer.go

```diff
@@ -591,18 +591,33 @@ func GetTimestamp(xpath *xpath.Expression, xmlNode xml.Node) *int64 {
 
 func TimestampToSeconds(timestamp string) *int64 {
 	var desiredDateUnix = new(int64)
-	year, _ := strconv.ParseInt(timestamp[0:4], 10, 32)
-	month, _ := strconv.ParseInt(timestamp[4:6], 10, 32)
-	day, _ := strconv.ParseInt(timestamp[6:8], 10, 32)
+	if len(timestamp) < 8 {
+		return nil
+	}
+	year, err := strconv.ParseInt(timestamp[0:4], 10, 32)
+	month, err := strconv.ParseInt(timestamp[4:6], 10, 32)
+	day, err := strconv.ParseInt(timestamp[6:8], 10, 32)
+	if err != nil {
```

Comment:
Does this use of `err` in 3 function return statements in a row mean that `err` will be overwritten each time? E.g. if the year parser breaks, but the day doesn't, what will be the value of `err`? The error from the year parser, or `nil` from the day parser?